projects/core_presentation.md

```diff
@@ -0,0 +1,51 @@
+Core Project Presentations
+==========================
+
+In order to increase the continuous learning of our team, and provide fodder for our marketing, we encourage Core Project Presentations
+
+At this point, there is no Template for such a presentation, but it should include the following:
+
+_NOTE: Total presentation should be between 50 minutes and 90 minutes.  Depending on the project, more time may be spent in one section than the other, but each should be considered._
+
+## Quick History and Context _(5-10 minutes)_
+
+* What was the motivation for initiating the project?
+* What is the hypothesis we’re trying to prove?
```

Comment:
I'm assuming that kind of stuff would be in the "motivation"... e.g. `Said Customer came to RoleModel seeking ...`